WordPress/WordPressTest/RemoteTestCase.swift

```diff
@@ -0,0 +1,61 @@
+import Foundation
+import XCTest
+import OHHTTPStubs
+@testable import WordPress
+
+class RemoteTestCase: XCTestCase {
+
+    // MARK: - Constants
+
+    let contentTypeJson     = "application/json"
+    let contentTypeJS       = "text/javascript;charset=utf-8"
+    let contentTypeHTML     = "application/html"
+    let timeout             = TimeInterval(1000)
+
+    // MARK: - Properties
+
+    var restApi: WordPressComRestApi!
+    var contextManager: TestContextManager!
```

Comment:
I'm wondering if calling this `var testContextManager` would be better so superclasses don't look like they're trying to use the default one based upon first glance of the name.